packages/bundler-security/src/constants.js

```diff
@@ -0,0 +1,12 @@
+const TREZOR_PACKAGES_SCOPES = ['@trezor', '@suite-common', '@suite-native'];
+const PACKAGES_PATHS = ['/packages/', '/suite-common/', '/suite-native/'];
```

Comment:
What if some node_module have another 3rd party dependency that also use `/packages` folder?

Reply:
It will break for sure, but I don't think will happen and we add whitelist then or something but I think it would premature optimalization now